azdev/operations/linter/__init__.py

```diff
@@ -158,8 +160,28 @@ def run_linter(modules=None, rule_types=None, rules=None, ci_exclusions=None,
         run_commands=not rule_types or 'commands' in rule_types,
         run_command_groups=not rule_types or 'command_groups' in rule_types,
         run_help_files_entries=not rule_types or 'help_entries' in rule_types)
+    print(os.linesep + 'Run custom pylint rules.')
```

Comment:
Will display or some azdev-builtin functions work instead of formating via raw print?